aten/src/ATen/native/Resize.h

```diff
@@ -94,6 +94,11 @@ inline void setStrided(
 
   /* size and stride */
   AT_ASSERT(size.size() == stride.size());
+
+  for(auto val : stride){
+    TORCH_CHECK(val >= 0, "Negative strides are not supported at the moment");
```

Comment:
nit: It would be nice if this error message (1) said which operator it is coming from and (2) told the user where the negative stride is. For example, 

```
TORCH_CHECK(val >= 0, "as_strided: Negative strides are not supported at the moment, got strides: ", strides);
```